app/views/users/new.html.erb

```diff
@@ -1,3 +1,23 @@
 <% provide :title, t("static_pages.signup.title") %>
 <h1><%= t "static_pages.signup.title" %></h1>
-<p><%= t "static_pages.signup.content" %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
+      <%= render "shared/error_messages" %>
+
+      <%= f.label t("users.new.name") %>
```

Comment:
k phải như này đâu a ơi
vẫn là 
`<%= f.label :name %>` 
nhưng có thể dùng i18n for label đc. A đọc lại tài liệu i18n trên viblo á a